pyproject.toml

```diff
@@ -95,6 +95,8 @@ all = [
 doc = [
     "Sphinx==8.0.2",
     "ansys-sphinx-theme==1.0.8",
+    "Jinja2 ==3.1.4",

```

Comment:
What issue did you face with sphinx-jinja?

Reply:
Mostly table rendering problems and for-loops inside the toctree not working. I took as reference many of the repos we have where we programatically reject sections (examples, API) but it still didn't work

Reply:
Oh! I see. Tables are tricky. Did you try going with a CSV format? This may be easier: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html#csv-files

Reply:
Aha! No, I did not. I went with the raw rST format for tables:

```rst
+-------+-------+
| A     | B     |
+=======+=======+
| a     | b     |
+-------+-------+
| a     | b     |
+-------+-------+
| a     | b     |
+-------+-------+
```

I can give it a try to CSV style but now that the logic is there, I'd always take prefer the native format rather than any other format that then gets adapted.

Reply:
Decided to leave it native format for better compatibility.